load-tests/route_guide_bidirectional_streaming/results/summary.csv

```diff
@@ -0,0 +1 @@
+Label,# Samples,Average,Median,90% Line,95% Line,99% Line,Min,Max,Error %,Throughput,Received KB/sec,Std. Dev.,Date,Payload,Users
```

Comment:
Seems this is a generated CSV. Can't we add a trailing newline when this is being generated?

Reply:
Added in https://github.com/ballerina-platform/module-ballerina-grpc/pull/490/commits/4735ce77a9784ecdac578c36813759626a0f4a37